core/src/main/java/nl/nn/adapterframework/configuration/ConfigurationDigester.java

```diff
@@ -221,18 +222,23 @@ private void digestConfiguration(Configuration configuration) throws Configurati
 	 * Performs an Identity-transform, which resolves entities with content from files found on the ClassPath.
 	 * Resolve all non-attribute properties
 	 */
-	public void parseAndResolveEntitiesAndProperties(ContentHandler digester, Configuration configuration, Resource resource, Properties appConstants) throws IOException, SAXException, TransformerConfigurationException {
+	public void parseAndResolveEntitiesAndProperties(ContentHandler digester, Configuration configuration, Resource resource, PropertyLoader properties) throws IOException, SAXException, TransformerConfigurationException {
 		ContentHandler handler;
 
 		XmlWriter loadedHiddenWriter = new XmlWriter();
 		handler = new PrettyPrintFilter(loadedHiddenWriter);
-		handler = new AttributePropertyResolver(handler, appConstants, getPropsToHide(appConstants));
+		handler = new AttributePropertyResolver(handler, properties, getPropsToHide(properties));
 		handler = new XmlTee(digester, handler);
 
-		handler = getStub4TesttoolContentHandler(handler, appConstants);
+		handler = getStub4TesttoolContentHandler(handler, properties);
 		handler = getConfigurationCanonicalizer(handler);
-		handler = new OnlyActiveFilter(handler, appConstants);
-		handler = new ElementPropertyResolver(handler, appConstants);
+		handler = new OnlyActiveFilter(handler, properties);
+		handler = new ElementPropertyResolver(handler, properties);
+
+		boolean rewriteLegacyClassNames = properties.getBoolean("migration.rewrite.legacyClassNames", false);
```

Comment:
NB: I'm open to suggestions for a better property-name than `migration.rewrite.legacyClassNames`

Reply:
Ik zou `migration.rewriteLegacyClassNames` gebruiken, dan kan alles (mocht er meer bij komen) onder het migration kopje vallen. Verder geen voorkeur.